index.html

```diff
@@ -11,69 +11,339 @@
 </head>
 
 <body>
-    <header>
+    <section class="toolbar">
+        <div class="nav-menu">
+            <a href="#">Welcome</a>
+            <div class="link-list">
+                <a href="#">project</a>
+                <a href="#">About</a>
+                <a href="#">Information</a>
+            </div>
+            <a href="#" class="menu-icon">
+                <i class="fa fa-bars"></i></a>
+        </div>
 
-        <section class="toolbar">
-            <div class="nav-menu">
-                <a href="#">Welcome</a>
-                <div class="link-list">
-                    <a href="#">project</a>
-                    <a href="#">About</a>
-                    <a href="#">Information</a>
-                </div>
-                <a href="#" class="menu-icon">
-                    <i class="fa fa-bars"></i></a>
+        <section class="headline">
+            <div class="primary-text">
+                <h1>Hey there, I'm Addis</h1>
+                <h4>I'm software Engineer</h4>
+            </div>
+            <div class="supporting-text">
+                <p>I develop websites and other software products with high quality.
+                    I have a series of experiences in building software for different customers. If you have any
+                    project, please contact me.
+                </p>
+                <p>below my contact</p>
+            </div>
+            <div class="social-media">
+                <ul class="social-nav">
+                    <li><a href="http://linkedin.com/AddisTsega"><i class="fa fa-linkedin"></i></a></li>
```

Comment:
Great work @Aseadd 🤝

- [ ] [Optional] You may please try to add that arrow in your design and space your elements according to the Figma design
- Figma design
![Figmas](https://user-images.githubusercontent.com/63917725/177014330-dcd16dcb-13b9-4225-94b2-c018b35f79aa.PNG)